client/src/components/EditBudget.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const EditBudget = (sum) => {
+  const [value, setValue] = useState(sum.budget);
+
+  const handleClick = () => {
+    sum.handleSaveClick(value);
+  };
+
+  const handleChange = (event) => {
+    setValue(event.target.value);
+  };
+
+  return (
+    <>
+      <Input
```

Comment:
You could consider adding a label in addition to the placeholder, so that it's easier for people using screen readers to fill in your form.